pkg/monitoring/metrics.go

```diff
@@ -33,10 +33,15 @@ var (
 			Name: "kubevirt_cnao_cr_kubemacpool_deployed",
 			Help: "Kubemacpool is deployed by Cnao CR",
 		})
+	nmstateHandlerDeployed = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "kubevirt_cnao_nmstate_deployed",
```

Comment:
hmmm.. I don't get something..
if you already defined `kubevirt_cnao_nmstate_deployed` with `kube_daemonset_labels`, i.e. without the use of a GAUGE variable, why do you need to define one here? isn't this redundant?

Reply:
Removed the variable in this [force-push](https://github.com/kubevirt/cluster-network-addons-operator/compare/faf97e0175d2f1fa3bd071b60a3f44fcccd668ca..7cc8bcd687897940a33fddda5cb0a38b1f71031a).